examples/README.md

```diff
@@ -4,11 +4,8 @@ The following examples are available in [the openforcefield toolkit repository](
 
 ### Index of provided examples
 
-* `SMIRNOFF_comparison/` - compare molecule energies from SMIRNOFF/OpenMM and AMBER
 * `SMIRNOFF_simulation/` - simulation of a molecule in the gas phase with the SMIRNOFF forcefield format
-* `chemicalEnvironments/` - example and documentation of using chemical environment objects to manipulate environment being considered, generate example SMIRKS, etc. Also contains IPython notebook using the chemical environment for depiction.
-* `forcefield_modification/` - Jupyter notebook example of modifying a forcefield parameter and evaluating how it changes an energy
-* `host_guest_simulation/` - example of setting up a basic MD simulation for a host-guest system using the SMIRNOFF force fields.
-* `label_molecule/` - example showing how to check which parameters are used in which molecules, do parameter usage statistics, etc.
-* `mixedFF_structure/` - example of how to use SMIRNOFF parameters for small molecules in combination with more conventional force fields for proteins and other components of your system (using ParmEd to combine parameterized structures)
-* `partial_bondorder/` - example of using partial bond orders for parameterizing molecules, and showing how the SMIRNOFF format extends; see https://github.com/openforcefield/smarty/issues/53
+* `forcefield_modification/` - modify forcefield parameters and evaluate how system energy changes
+* `conversion_amber_gromacs` - convert a System generated with the Open Forcefield Toolkit, which can be simulated natively with OpenMM, into AMBER prmtop/inpcrd and GROMACS top/gro input files through the ParmEd library.`
```

Comment:
Suggest `using_smirnoff_in_amber_or_gromacs`

Reply:
Good call. Changed in eaaf7db